dpnp/dpnp_iface_trigonometric.py

```diff
@@ -929,10 +932,16 @@ def sqrt(x1):
     """
 
     x1_desc = dpnp.get_dpnp_descriptor(x1, copy_when_strides=False, copy_when_nondefault_queue=False)
-    if x1_desc:
-        return dpnp_sqrt(x1_desc).get_pyobj()
+    if x1_desc and not kwargs:
```

Comment:
I think it'd better not to call get_dpnp_descriptor() if `kwargs`, since it makes no sense.